src/ui/server/controllers/insurance/business/broker/index.ts

```diff
@@ -9,6 +9,7 @@ import generateValidationErrors from './validation';
 import mapAndSave from '../map-and-save/broker';
 import isCheckAndChangeRoute from '../../../../helpers/is-check-and-change-route';
 import { Request, Response } from '../../../../../types';
+import constructPayload from '../../../../helpers/construct-payload';
```

Comment:
Can we have the import in the order it's executed? Recently tidied up a lot of imports so that `types` is the last thing imported